<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Contrôles</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5"/>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boutonNB_">
        <property name="text">
         <string>Noir et Blanc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boutonGris_">
        <property name="text">
         <string>Gris</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boutonNegatif_">
        <property name="text">
         <string>Negatif</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Propriétées</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Valeurs</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>Nom:</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>Largeur:</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>Hauteur:</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="ImageScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>397</width>
         <height>483</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOuvrir_"/>
    <addaction name="actionEnregistrerSous_"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="undo_"/>
    <addaction name="redo_"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAPropos_"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionEnregistrer_Sous">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Enregistrer Sous</string>
   </property>
  </action>
  <action name="action_Propos">
   <property name="text">
    <string>À Propos</string>
   </property>
  </action>
  <action name="actionEnregistrer_le_groupe">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Enregistrer le groupe</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionOuvrir_">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionEnregistrerSous_">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="undo_">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="redo_">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionAPropos_">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
